buku

```diff
@@ -4658,9 +4667,10 @@ def prompt(obj, results, noninteractive=False, deep=False, listtags=False, sugge
             pass
         return
 
+    skip_print = False
     while True:
-        if new_results or nav == 'n':
-            count = 0
+        if (new_results or nav == 'n') and not skip_print:
+            count = next_index
```

Comment:
It makes more sense for the printed search result indices (`31.`, `32.`, `33.`) to match the page indices (`31-40/55`)